connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DB2IntegrationSuite.scala

```diff
@@ -217,4 +217,25 @@ class DB2IntegrationSuite extends DockerJDBCIntegrationSuite {
     assert(actual.length === 2)
     assert(actual.toSet === expectedResult)
   }
+
+  test("SPARK-42534: DB2 Limit pushdown test") {
+    val actual = sqlContext.read
+      .format("jdbc")
+      .option("url", jdbcUrl)
+      .option("dbtable", "tbl")
+      .load()
+      .limit(2)
```

Comment:
Limit push down is only supported in V2 JDBC. I don't think this `limit(2)` will trigger the newly added `DB2Dialect.getLimitClause`. This test should be done in `v2/DB2IntegrationSuite`

Reply:
There are already tests covering limit pushdown.

Reply:
I think it's OK to leave the test here, in case one day we support limit pushdown in v1 JDBC as well.